### PR TITLE
Code review!

### PR DESCRIPTION
**Pull request by shaliqnigal**
This branch is created in order to assign a reviewer.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+version: 2.1
```
Comment (VenkataNagaShanmukh-Chava) [inline .circleci/config.yml:1]: Squash all update config file to single commit

File: test-api-js/todo.service.js [hunk 2]
```diff
@@ -24,16 +24,31 @@ class todoservice{
 
     add_todo(todo){
         // Your code here
+        this.todo_data.todo.push(todo);
+        console.log(this.todo_data.todo);
```
Comment (VenkataNagaShanmukh-Chava) [inline test-api-js/todo.service.js:28]: I think console.log can be removed as we are merging to master

File: test-api-js/todo.service.js [hunk 3]
```diff
@@ -24,16 +24,31 @@ class todoservice{
...
+        return this.todos;
     }
 
     delete_todo(id){
         // Your code here
+        this.todo_data.todo.splice(id-1,1)
```
Comment (VenkataNagaShanmukh-Chava) [inline test-api-js/todo.service.js:34]: we can have edge case where user passes the id is not in array for delete and update

**Review by VenkataNagaShanmukh-Chava — COMMENTED** [review]
We can have gitignore file to not add node modules to branch.
